definitions/npm/faker_v4.x.x/flow_v0.25.x-/faker_v4.x.x.js

```diff
@@ -0,0 +1,261 @@
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      animals: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      business: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      cats: (width?: ?number, height?: ?number, randomize?: boolean) => string,
+      city: (width?: ?number, height?: ?number, randomize?: boolean) => string,
+      food: (width?: ?number, height?: ?number, randomize?: boolean) => string,
+      nightlife: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      fashion: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      people: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      nature: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      sports: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      technics: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      transport: (
+        width?: ?number,
+        height?: ?number,
+        randomize?: boolean
+      ) => string,
+      dataUri: (width?: ?number, height?: ?number) => string
+    },
+    internet: {
+      avatar: () => string,
+      email: (
+        firstName?: ?string,
+        lastName?: ?string,
+        provider?: ?string
+      ) => string,
+      exampleEmail: (firstName?: ?string, lastName?: ?string) => string,
+      userName: (firstName?: ?string, lastName?: ?string) => string,
+      protocol: () => string,
+      url: () => string,
+      domainName: () => string,
+      domainSuffix: () => string,
+      domainWord: () => string,
+      ip: () => string,
+      ipv6: () => string,
+      userAgent: () => string,
+      color: (
+        baseRed255?: ?number,
+        baseGreen255?: ?number,
+        baseBlue255?: ?number
+      ) => string,
+      mac: () => string,
+      password: (
+        len?: ?number,
+        memorable?: ?boolean,
+        pattern?: ?string,
+        prefix?: ?string
+      ) => string
+    },
+    lorem: {
+      word: (count?: number) => string,
+      words: (count?: number) => string,
+      sentences: (sentenceCount?: ?number, separator?: string) => string,
+      slug: (wordCount?: number) => string,
+      sentence: (wordCount?: ?number, range?: number) => string,
+      paragraph: (sentenceCount?: number) => string,
+      paragraphs: (paragraphCount?: ?number, separator?: string) => string,
+      text: (times?: number) => string,
+      lines: (lineCount?: number) => string
+    },
+    name: {
+      firstName: (gender?: mixed) => string,
+      lastName: (gender?: mixed) => string,
+      findName: (
+        firstName: string,
```

Comment:
this should be `firstName?: ?string,`, because it's optional

Reply:
Huh. The docs indicate it's required: https://marak.github.io/faker.js/faker.name.html#-static-findName__anchor, but the code shows that its not: https://github.com/Marak/faker.js/blob/3a4bb358614c1e1f5d73f4df45c13a1a7aa013d7/lib/name.js#L68-L77

Thanks! Will change.

Reply:
Done.